index.html

```diff
@@ -22,6 +22,18 @@
     </nav>
   </header>
 
+  <!-- Main Page Content -->
+
```

Comment:
Move image inside content div and wrap p tags in additional div for layout organization.